lib/chef_zero/endpoints/acls_endpoint.rb

```diff
@@ -22,6 +22,17 @@ def get(request)
         end
         acls = FFI_Yajl::Parser.parse(get_data(request, acl_path), :create_additions => false)
         acls = ChefData::DataNormalizer.normalize_acls(acls)
+        if request.query_params["detail"] == "granular"
+          acls.each do |perm, ace|
+            acls[perm]["actors"] = []
```

Comment:
Out of curiosity - does it make sense to leave all the actors in here even if granular is requested?